### ISSUE TITLE
Prevent roster movements when a team's season is over

### ISSUE THREAD

**Issue by mdryden** [mdryden/110yards#23]
**Is your feature request related to a problem? Please describe.**
It's possible for roster players to be made by a team who's season is over, potentially disrupting players who are still playing.

**Describe the solution you'd like**
Rosters should be frozen when there are no current/future matchups scheduled.

**Comment by kamikazecanuck** [comment]
I don’t necessarily mind that. If you in a keeper league that’s the best time for the bottom feeders to pick up players.

**Comment by mdryden** [comment]
That's true, but I think I'd still lock them until the season is over for
everyone.  I wouldn't want eliminated teams messing with teams still in
play, either by picking up players or by dropping players and messing with
balance.

**Comment by mdryden** [comment]
I could make it a league option though.

**Comment by kamikazecanuck** [comment]
I don’t know how much you can really mess with people. When you’re eliminated that’s the time to pick up promising or superstar injured players. People still in the game are concentrating on their matchup so it kind of works itself out as the worst teams are getting first dibs. Plus if your QB or K was on a bye in your last game you’ll need a control of your team to sort it out again after the week resolved. Even if it’s just to release them for teams that are still in the playoffs. So could actually have opposite effect.

**Comment by mdryden** [comment]
That's fair.  I've seen it happen though...one guy gets knocked out and he
drops his best players for his buddy to pick up for his matchup.  Obviously
easily resolved by commissioner control, but easily avoided too.

You've sold me on making it an option though.

**Comment by kamikazecanuck** [comment]
Actually, I think I'm coming around to your way of thinking.  You can't use Commissioner controls to rectify any shenanigans because the teams that are out aren't visible to the Commissioner.  
I'm not sure what the solution would be because I think you should be able to cut down players for end year state.  Maybe knocked out players are locked until the season is over?  Anyway if its an option I'll probably put it to a vote.

**Comment by kamikazecanuck** [comment]
Ok after some mild shenanigans this year an idea was put out there: how about eliminated players can only drop players during the waiver period. Probably hard to code for but could be a good middle ground.

**Comment by mdryden** [comment]
Yea that could work.  I'll think on it.

On Oct. 12, 2018 9:58 p.m., "kamikazecanuck" <notifications@github.com>
wrote:

Ok after some mild shenanigans this year an idea was put out there: how
about eliminated players can only drop players during the waiver period.
Probably hard to code for but could be a good middle ground.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/mdryden/110yards/issues/23#issuecomment-429505775>, or mute
the thread
